### PR TITLE
added form

### PR DESCRIPTION
**Pull request by tatiana-sadaeva**
[DEMO LINK](https://tatiana-sadaeva.github.io/layout_html-form/)
[TEST REPORT LINK](https://tatiana-sadaeva.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,17 +1,175 @@
...
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <script type="text/javascript" src="./main.js">
+    </script>
+    <form action="http://localhost:3000/api" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input type="text"
```
[the remark this answers is not in the document]
Comment (agniya-i) [reply src/index.html:22]: start first attribute also on new line and fix it everywhere

File: src/index.html [hunk 2]
```diff
@@ -1,17 +1,175 @@
...
+            min="1" 
+            max="100"
+            value="12"
+          >
+        </label>
+
+
+        <label class="form-field">
+          Full date of birth:
+          <input type="date"
```
Comment (agniya-i) [inline src/index.html:48]:
```suggestion
          <input 
            type="date" 
```

File: src/index.html [hunk 3]
```diff
@@ -1,17 +1,175 @@
...
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          <label>
+          Do you love cats?
+            <input type="radio" 
+              name="cats" 
+              value="m" 
+            > Yes
```
Comment (agniya-i) [inline src/index.html:100]:
```suggestion
            > 
            Yes
```

File: src/index.html [hunk 4]
```diff
@@ -1,17 +1,175 @@
...
+          </label>
+
+          <label>
+            <input type="radio" 
+              name="cats" 
+              value="f"
+            > No
```
Comment (agniya-i) [inline src/index.html:107]:
```suggestion
            > 
            No
```

File: src/index.html (hunk 3 above)
Comment (agniya-i) [reply src/index.html:100]: move on new line

File: src/index.html [hunk 5]
```diff
@@ -1,17 +1,175 @@
...
+          >
+        </label>
+      </fieldset>
+
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          <label>
+          Do you love cats?
```
Comment (agniya-i) [inline src/index.html:96]: remove this text from input label, and move above `<label>`
```suggestion
          Do you love cats?
          <label>
```

File: src/index.html [hunk 6]
```diff
@@ -1,17 +1,175 @@
...
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+
+        <label class="form-field">
+          How do you rate our work?
+          <input type="range" 
+            name="cowbell" 
+            min="0" max="100"
```
Comment (agniya-i) [inline src/index.html:147]:
```suggestion
            min="0" 
            max="100" 
```

File: src/index.html [hunk 7]
```diff
@@ -1,17 +1,175 @@
...
+          <input type="text" 
+            name="surname" 
+            autocomplete="off"
+          >
+        </label>
+
+
+        <label class="form-field">
+          Name:
+          <input type="text" name="name">
```
Comment (agniya-i) [inline src/index.html:31]: text fields should have `autocomplete="off"`

**Review by agniya-i — CHANGES_REQUESTED** [review]

**Comment by tatiana-sadaeva** [comment]
[DEMO LINK](https://tatiana-sadaeva.github.io/layout_html-form/)
[TEST REPORT LINK](https://tatiana-sadaeva.github.io/layout_html-form/report/html_report/)

File: src/style.css [hunk 8]
```diff
@@ -1 +1,21 @@
-/* styles go here */
+.form-field {
+  display: block;
+  margin-bottom: 10px;
+  padding-bottom: 0;
```
Comment (SanyaBratashchuk) [inline src/style.css:4]:
```suggestion
  
```

File: src/style.css [hunk 9]
```diff
@@ -1 +1,21 @@
...
+  margin-bottom: 20px;
+}
+
+label:last-child {
+  margin-bottom: 0;
+}
+
+input.form-field {
+  margin-bottom: 5px;
+}
```
Comment (SanyaBratashchuk) [inline src/style.css:21]:
```suggestion
```

File: src/style.css [hunk 10]
```diff
@@ -1 +1,21 @@
...
+  margin-top: 10px;
+}
+
+fieldset {
+  margin-bottom: 20px;
+}
+
+label:last-child {
+  margin-bottom: 0;
+}
```
Comment (SanyaBratashchuk) [inline src/style.css:17]: use classes for styling purposes

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

**Pull request closed without merging**